### DISCUSSION TITLE
An Oh-My-Posh upgrade notice segment

### DISCUSSION THREAD

**Discussion by Lockszmith-GH** [JanDeDobbeleer/oh-my-posh#3840] [Ideas]
I couldn't find this by searching either the GitHub repo/community or the documentation.

If there isn't a suggestion like this, a segment that would show up when an upgrade is available, fuled by the state of the [upgrade notice](https://ohmyposh.dev/docs/faq#i-want-to-disable-the-upgrade-notice) would be a great addition.

**Comment by JanDeDobbeleer** [comment]
#3874

**Reply by Lockszmith-GH** [reply]
Thank you!
This is excellent. https://ohmyposh.dev/docs/segments/upgrade
I've added it, but now I need to wait for the next version to see it actually works :)

**Reply by Lockszmith-GH** [reply]
This is perfect. Thank you so much!

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/905716/810cc3bc-2344-46aa-80e5-2ee1557800a8)
